tor/core/user_interaction.py

```diff
@@ -112,9 +112,12 @@ def process_claim(
     elif response.status == BlossomStatus.blacklisted:
         message = i18n["responses"]["general"]["blacklisted"]
 
-    else:
+    elif response.status == BlossomStatus.already_claimed:
         message = i18n["responses"]["claim"]["already_claimed"]
 
+    else:
+        message = i18n["responses"]["general"]["oops"]
+
```

Comment:
Does it give us a Slack ping in this case? If not, I think it should

Reply:
It doesn't give us a ping, but this would be a great thing to add. Maybe a bugsnag hit instead? Either way, it's outside the scope of this.